python/src/etos_api/__init__.py

```diff
@@ -55,7 +55,7 @@
     trace.set_tracer_provider(PROVIDER)
 
     FastAPIInstrumentor().instrument_app(
-        APP, tracer_provider=PROVIDER, excluded_urls="selftest/.*,logs/.*"
+        APP, tracer_provider=PROVIDER, excluded_urls=".*/etos/ping"
```

Comment:
Won't this make all the requests to selftest show up in OTEL signals? Sure there is a redirect but the request from the older clients are still to the older urls, or have I got it all backwards?

Reply:
Update so that the deployment healthcheck uses the new endpoint. The "selftest/ping" endpoint will be used by old clients giving us a heads-up.